plugins/connection/pct_remote.py

```diff
@@ -0,0 +1,772 @@
+# -*- coding: utf-8 -*-
+# Derived from ansible/plugins/connection/paramiko_ssh.py (c) 2012, Michael DeHaan <michael.dehaan@gmail.com>
+# Copyright (c) 2024 Nils Stein (@mietzen) <github.nstein@mailbox.org>
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (annotations, absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+    author: Nils Stein (@mietzen) <github.nstein@mailbox.org>
+    name: pct_remote
+    short_description: Run tasks in Proxmox LXC container instances using pct CLI via SSH
+    requirements:
+      - paramiko
+    description:
+      - Run commands or put/fetch files to an existing Proxmox LXC container using pct CLI via SSH.
+      - Use the Python SSH implementation (Paramiko) to connect to Proxmox.
+    version_added: "10.1.0"
```

Comment:
I am writing 10.2.0, but it is set to be released in the next couple of days, maybe wait for that and set this to the next one.
```suggestion
    version_added: "10.2.0"
```

Reply:
Done